docs/tuf-spec.txt

```diff
@@ -1008,6 +1008,23 @@ Version 1.0 (Draft)
    ensure that outdated clients remain able to update, without requiring all 
    previous root keys to be kept to sign new root.json metadata.
 
+   The exact steps to perform the root.json migration should be as follows:
+     
+   1. The client starts at version N of root.json
+
+   2. The client downloads version N + 1 of root.json
+
+   3. The client uses the keys and threshold defined in version N to
+      validate the signatures of N + 1
+
+   4. The client flushes all knowledge of root.json verion N then uses the
+      updated keys and threshold to validate the signatures of N + 1
```

Comment:
@vladimir-v-diaz You're right. I should say the flush happens *after* the validation.